### ISSUE TITLE
HF ATen operations

### ISSUE THREAD

**Issue by EikanWang** [intel/torch-xpu-ops#68]
We have landed the first PR to support XPU ATen operations gradually. At the moment, PyTorch Dynamo HF eager mode is the current priority. It contains >40 models and > 150 operations. Regarding these operations, we will enable these operations gradually.

- [ ]  `aten::_convolution`
- [ ] #74
- [ ]  `aten::_foreach_add_`
- [ ]  `aten::_index_put_impl_`
- [ ]  `aten::_local_scalar_dense`
- [ ]  `aten::_log_softmax`
- [ ]  `aten::_log_softmax_backward_data`
- [x]  `aten::_reshape_alias`
- [ ]  `aten::_softmax`
- [ ]  `aten::_softmax_backward_data`
- [x]  `aten::_to_copy`
- [ ]  `aten::_unsafe_view`
- [x]  `aten::abs`
- [x]  `aten::add`
- [x]  `aten::add_`
- [ ]  `aten::addmm`
- [ ]  `aten::alias`
- [ ]  `aten::any`
- [x]  `aten::arange`
- [ ]  `aten::argmax`
- [x]  `aten::as_strided`
- [x]  `aten::as_strided_`
- [ ]  `aten::bernoulli_`
- [x]  `aten::bitwise_not`
- [ ]  `aten::bmm`
- [ ]  `aten::cat`
- [ ]  `aten::chunk`
- [x]  `aten::clamp`
- [x]  `aten::clamp_min`
- [x]  `aten::clone`
- [ ]  `aten::col2im`
- [ ]  `aten::constant_pad_nd`
- [ ]  `aten::contiguous`
- [ ]  `aten::conv1d`
- [ ]  `aten::convolution`
- [ ]  `aten::convolution_backward`
- [x]  `aten::copy_`
- [x]  `aten::cos`
- [ ]  `aten::cross_entropy_loss`
- [ ]  `aten::cudnn_convolution`
- [ ]  `aten::cumsum`
- [ ]  `aten::detach`
- [ ]  `aten::detach_`
- [x]  `aten::div`
- [x]  `aten::div_`
- [ ]  `aten::divide_`
- [ ]  `aten::dropout`
- [ ]  `aten::einsum`
- [ ]  `aten::embedding`
- [ ]  `aten::embedding_backward`
- [ ]  `aten::embedding_dense_backward`
- [x]  `aten::empty`
- [x]  `aten::empty_like`
- [x]  `aten::empty_strided`
- [x]  `aten::eq`
- [ ]  `aten::expand`
- [ ]  `aten::expand_as`
- [ ]  `aten::fft_fftn`
- [x]  `aten::fill_`
- [ ]  `aten::flatten`
- [ ]  `aten::flip`
- [ ]  `aten::full`
- [ ]  `aten::full_like`
- [ ]  `aten::gather`
- [ ]  `aten::gelu`
- [ ]  `aten::gelu_backward`
- [x]  `aten::gt`
- [ ]  `aten::im2col`
- [ ]  `aten::index`
- [ ]  `aten::index_add_`
- [ ]  `aten::index_select`
- [ ]  `aten::index_select_backward`
- [ ]  `aten::is_nonzero`
- [ ]  `aten::item`
- [ ]  `aten::layer_norm`
- [x]  `aten::le`
- [ ]  `aten::lift_fresh`
- [ ]  `aten::linear`
- [x]  `aten::log`
- [ ]  `aten::log_softmax`
- [x]  `aten::lt`
- [ ]  `aten::masked_fill`
- [ ]  `aten::masked_fill_`
- [ ]  `aten::matmul`
- [ ]  `aten::max`
- [ ]  `aten::maximum`
- [ ]  `aten::mean`
- [ ]  `aten::min`
- [ ]  `aten::minimum`
- [ ]  `aten::mm`
- [x]  `aten::mul`
- [ ]  `aten::multiply`
- [ ]  `aten::narrow`
- [ ]  `aten::native_dropout`
- [ ]  `aten::native_dropout_backward`
- [ ]  `aten::native_layer_norm`
- [ ]  `aten::native_layer_norm_backward`
- [x]  `aten::ne`
- [x]  `aten::neg`
- [ ]  `aten::new_empty`
- [ ]  `aten::new_empty_strided`
- [ ]  `aten::new_ones`
- [ ]  `aten::new_zeros`
- [ ]  `aten::nll_loss`
- [ ]  `aten::nll_loss_backward`
- [ ]  `aten::nll_loss_forward`
- [ ]  `aten::nll_loss_nd`
- [x]  `aten::ones`
- [x]  `aten::ones_like`
- [ ]  `aten::pad`
- [ ]  `aten::permute`
- [x]  `aten::pow`
- [ ]  `aten::rand`
- [ ]  `aten::real`
- [ ]  `aten::reciprocal`
- [x]  `aten::relu`
- [ ]  `aten::remainder`
- [ ]  `aten::repeat`
- [x]  `aten::reshape`
- [x]  `aten::resize_`
- [ ]  `aten::result_type`
- [x]  `aten::rsqrt`
- [x]  `aten::rsub`
- [ ]  `aten::scalar_tensor`
- [ ]  `aten::select`
- [ ]  `aten::select_backward`
- [x]  `aten::sin`
- [ ]  `aten::slice`
- [ ]  `aten::slice_backward`
- [ ]  `aten::softmax`
- [ ]  `aten::split`
- [x]  `aten::sqrt`
- [ ]  `aten::squeeze`
- [x]  `aten::sub`
- [ ]  `aten::sum`
- [ ]  `aten::t`
- [x]  `aten::tanh`
- [x]  `aten::tanh_backward`
- [x]  `aten::threshold_backward`
- [x]  `aten::to`
- [x]  `aten::transpose`
- [ ]  `aten::tril`
- [ ]  `aten::type_as`
- [x]  `aten::unfold`
- [x]  `aten::uniform_`
- [ ]  `aten::unsqueeze`
- [x]  `aten::view`
- [x]  `aten::view_as_complex`
- [x]  `aten::view_as_real`
- [x]  `aten::where`
- [x]  `aten::zero_`
- [x]  `aten::zeros`
- [x]  `aten::zeros_like`

**Comment by riverliuintel** [comment]
Done.
